holidays/countries/mauritius.py

```
@@ -0,0 +1,203 @@
+    # https://web.archive.org/web/20240910225221/https://www.timeanddate.com/holidays/mauritius/thaipoosam-cavadee
+    THAIPUSAM_DATES = {
+        2020: (FEB, 8),
+        2021: (JAN, 28),
+        2022: (JAN, 18),
+        2023: (FEB, 4),
+        2024: (JAN, 25),
+        2025: (FEB, 11),
+    }
+
+
+class MauritiusIslamicHolidays(_CustomIslamicHolidays):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)